org.js.nuclear.Nuclear.json

```diff
@@ -0,0 +1,67 @@
+{
+    "app-id": "org.js.nuclear.Nuclear",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "branch": "stable",
+    "sdk": "org.freedesktop.Sdk",
+    "separate-locales": false,
+    "command": "run.sh",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--filesystem=host",
+	"--socket=pulseaudio"
+    ],
+    "build-options" : {
+        "env": {
+            "NPM_CONFIG_LOGLEVEL": "info"
+        }
+    },
+    "modules": [
+        {
+            "name": "nuclear",
+            "buildsystem": "simple",
+            "sources": [
+               {
+                   "type": "file",
+                   "url": "https://github.com/nukeop/nuclear/releases/download/v0.6.3/nuclear-fca030.tar.gz",
+                   "sha256": "5b90a1064736c158f62ea74c5814dffdd25f8b6a3877dfeed065377e744cdd2e",
+                   "dest-filename":"nuclear-fca030.tar.gz"
+                },
+                {
+                    "type":"file",
+                    "path": "org.js.nuclear.Nuclear.metainfo.xml"
+                },
+                {
+                  "type":"file",
+                  "path": "org.js.nuclear.Nuclear.desktop"
+                },
+                {
+                    "type":"file",
+                    "path":"org.js.nuclear.Nuclear.png"
+                },
+                {
+                    "type":"file",
+                    "url":"https://raw.githubusercontent.com/advaithm/flathub/nuclear/build_for_arch.py",
```

Comment:
again, you want `path` not `url` since it is already here

Reply:
yeah i forgot. fist time it ran i had the path thing. it failed.